satellite/satellitedb/repairqueue_test.go

```diff
@@ -48,3 +53,78 @@ func TestRepairQueue(t *testing.T) {
 		}
 	})
 }
+
+func TestRepairQueueOrder(t *testing.T) {
+	satellitedbtest.Run(t, func(t *testing.T, db satellite.DB) {
+		ctx := testcontext.New(t)
+		defer ctx.Cleanup()
+
+		repairQueue := db.RepairQueue()
+
+		nullPath := []byte("/path/null")
+		recentRepairPath := []byte("/path/recent")
+		oldRepairPath := []byte("/path/old")
+		olderRepairPath := []byte("/path/older")
+
+		for _, path := range [][]byte{oldRepairPath, recentRepairPath, nullPath, olderRepairPath} {
+			injuredSeg := &pb.InjuredSegment{Path: path}
+			err := repairQueue.Insert(ctx, injuredSeg)
+			require.NoError(t, err)
+		}
+
+		dbAccess := db.(interface{ TestDBAccess() *dbx.DB }).TestDBAccess()
+		var timeConvertPrefix string
+		switch d := dbAccess.DB.Driver().(type) {
+		case *sqlite3.SQLiteDriver:
+			timeConvertPrefix = "datetime("
```

Comment:
I've seen in some PRs that we're moving away from using `datetime` in sqlite due the precision that it has. Shouldn't this test be updated accordingly?

Reply:
I was not aware that we were moving away from datetime, but the repair queue select query itself uses datetime, so I wanted to be consistent here

Reply:
Moby the date time is UTC based? make sure the timing is handled correctly. Other than that test does what is explained in the PR.

Reply:
yeah, it should be utc based after I did some searching. These are also the same strings we use for postgres vs. sqlite elsewhere in the codebase